gravitee-apim-e2e/api-test/apis/api-quality.spec.ts

```diff
@@ -0,0 +1,519 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { forManagementAsAdminUser, forManagementAsApiUser, forManagementAsSimpleUser } from '@client-conf/*';
+import { afterAll, beforeAll, describe, expect } from '@jest/globals';
+import { PortalApi } from '@management-apis/PortalApi';
+import { ConfigurationApi } from '@management-apis/ConfigurationApi';
+import { UsersApi } from '@management-apis/UsersApi';
+import { SearchableUser } from '@management-models/SearchableUser';
+import * as _ from 'lodash';
```

Comment:
We should import function directly instead of the whole `lodash`

Reply:
:+1: done